src/lavinmq/mfile.cr

```diff
@@ -168,7 +168,8 @@ class MFile < IO
   def copy_to(output : IO, size = @size) : Int64
     if unmapped? # don't remap unmapped files
       raise ClosedError.new if @fd < 0
-      io = IO::FileDescriptor.new(@fd, blocking: true, close_on_finalize: false)
+      io = IO::FileDescriptor.new(@fd, close_on_finalize: false)
+      IO::FileDescriptor.set_blocking(@fd, true)
```

Comment:
I wonder if we should put `{% if flag?(:unix) %}` around this?
In the comment for the method it says `NOTE: Only implemented on UNIX targets. Raises on Windows.`. Dunno why that isn't in the api docs.

Reply:
Hmm, maybe.. It's very much not the only thing that doesn't work on Windows though 🙂